src/modules/display.js

```diff
@@ -0,0 +1,34 @@
+export default class ControlDisplay {
```

Comment:
Your application is working fine except for not rendering the data in your local storage after refreshing the page. Please make the necessary changes to correct this error.

Check the image below for a better understanding:
1. Image before refreshing the page, the data is saved in your local storage and rendered.
![Screenshot from 2022-01-20 12-29-03](https://user-images.githubusercontent.com/75709904/150391595-3e55e12f-8d8b-4f9e-91cd-ed6214dca7df.png)

2. Image after refreshing your page, the data is still saved in your local storage but is not rendered.
![Screenshot from 2022-01-20 12-30-02](https://user-images.githubusercontent.com/75709904/150391617-88756a25-016b-4d69-bb5b-a6b6ed1fa545.png)